app/src/main/java/de/tum/in/tumcampusapp/component/ui/cafeteria/details/OpenHoursHelper.kt

```diff
@@ -81,22 +83,35 @@ class OpenHoursHelper(private val context: Context) {
             // Check the relativity
             val relativeTo: DateTime
             val relation: Int
+
+            val res: String
+            val relativeTime: String
+
             if (opens.isAfter(now)) {
                 relation = R.string.opens
                 relativeTo = opens
+                relativeTime = DateTimeUtils.formatFutureTime(relativeTo, context)
+                res = context.getString(relation) + " " + relativeTime.substring(0, 1)
```

Comment:
I dont like this part of the code as it looks kind of spaggetti to me.

Changes I would like:
- figgure out if relativeTime.substring(0, 1) is really needed. I dont see how anything but a number can be in this spot without looking deeper into this. Adding a comment for this would be appreciated
- I see why you put the return statement up here. 
  Inlining `relation` and  `relativeTo` and extracting a variable for common code would be a readability improvement imho
an example for this is for example the `relativeTime.substring(..)` part